app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -140,14 +141,14 @@
         filters.push({predicate: predicate, value: filter});
       }
 
-      _.each(searchTerms, function(term){
-        if(term.indexOf('is:') >=0) {
+      _.each(searchTerms, function(term) {
```

Comment:
sidetracking...

should we setup a linter/jshint? it could be tough to implement something like that this far in the project, but maybe we could do it file by file? everyone's been pretty great about sharing a similar style though.

maybe setup an .editorconfig?